LoginPlugin/iOS/ZappLoginPluginExample-iOS/FacebookInfoViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FacebookInfoViewController.swift
+//  ZappLoginPlugins
+//
+//  Created by Roi Kedarya on 17/05/2018.
+//  Copyright © 2018 Applicaster. All rights reserved.
+//
+
+import Foundation
+import ZappPlugins
+
+@objc class FacebookInfoViewController: UIViewController {
```

Comment:
can you add a comment with a few words about what this class is doing and its role?